src/app/screens/Dashboard/screens/Home/index.js

```diff
@@ -1,24 +1,22 @@
 import React, { useContext } from 'react';
+import { useQuery } from '@apollo/react-hooks';
 
 import { GlobalContext } from '~context/GlobalProvider';
+import { getAllComponentsByCategory } from '~utils/queries';
 
-import Card from './components/Card';
+import CardsContainer from './components/CardsContainer';
 import styles from './styles.module.scss';
-import { getComponentsCode } from './constants';
+import { getComponentsCode } from './utils';
 
 function Home() {
-  const COMPONENTS = getComponentsCode(); /* TODO: graphQL */
   const { state } = useContext(GlobalContext);
   const { category } = state;
+  const { loading, data } = useQuery(getAllComponentsByCategory(category));
+  const components = !loading && data ? getComponentsCode(data.repository.object.entries) : [];
```

Comment:
I think this is not necessary since you are wrapping your component with a HOC, meaning that it won't render the components until loading becomes false. 

```suggestion
  const components =  data ? getComponentsCode(data.repository.object.entries) : [];
```